unitTests/ui/DistanceResourcesTest.ts

```diff
@@ -155,6 +117,14 @@ export function DistanceResourcesTest() {
 
         expect(test.env.queryController.executeQuery).toHaveBeenCalledTimes(2);
       });
+
+      it('should send a search event triggered caused by the positionSet', () => {
```

Comment:
Hmm, should the test description be slightly modified to mention that we are sending an analytics event instead of a search event?